templates/kubernetes/config.yml

```diff
@@ -0,0 +1,5 @@
+name: Kubernetes
+description: |
+  Kubernetes service
+version: v0.1.0-rancher1
+category: Rancher System Services
```

Comment:
Older version of Rancher already filter out the category `System`, so we should use that (or the templates will show up in v1.0.x when published)